LICENSE.txt

```diff
@@ -74,6 +74,9 @@ File: skimage/feature/_fisher_vector.py
 Copyright: 2014 2014 Dan Oneata
 License: MIT
 
+File: skimage/transform/_thin_plate_splines.py
+Copyright: 2007 Zachary Pincus
+License: GNU General Public License v2.0
```

Comment:
This needs to be updated to use a compatible license which the original author allowed.

Reply:
No problem :) License declaration is here: https://github.com/scikit-image/scikit-image/issues/2429#issuecomment-816402127

Reply:
You can include that URL (https://gist.github.com/zpincus/be1e2293ec0d679b0f6d3e600ac8fccc) as reference, if you want.